samples/sample-app-storage/src/main/java/com/azure/android/storage/sample/MsalClient.java

```diff
@@ -0,0 +1,94 @@
+package com.azure.android.storage.sample;
+
+import android.app.Activity;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.azure.android.storage.sample.core.credential.AccessToken;
+import com.azure.android.storage.sample.core.util.tokenrequest.TokenResponseCallback;
+import com.microsoft.identity.client.AuthenticationCallback;
+import com.microsoft.identity.client.IAuthenticationResult;
+import com.microsoft.identity.client.PublicClientApplication;
+import com.microsoft.identity.client.exception.MsalClientException;
+import com.microsoft.identity.client.exception.MsalException;
+import com.microsoft.identity.client.exception.MsalServiceException;
+import com.microsoft.identity.client.exception.MsalUiRequiredException;
+
+import org.threeten.bp.DateTimeUtils;
+import org.threeten.bp.OffsetDateTime;
+import org.threeten.bp.ZoneOffset;
+
+import java.util.Date;
+
+class MsalClient {
```

Comment:
Nit: How do you feel about capitalizing this (and other MSAL-related classes) like `MSALClient`?

Reply:
Java guideline is to use pascal-case for acronyms so `Msal`.

Reply:
https://azure.github.io/azure-sdk/java_design.html#java-naming-uppercase-acronyms